theSrc/scripts/LabeledScatter.js

```diff
@@ -414,6 +443,90 @@ class LabeledScatter {
     return annotations
   }
 
+  adjustTitles (plotly_chart_layout, config) {
```

Comment:
Why does adjustTitles have to be called in the `after_plot` event handler? I would have thought that zooming or clicking on things wouldn't affect the adjusted titles?

Reply:
Plotly moves the title and annotation back to its original location after zooming, so we need to readjust the titles.